### PR TITLE
Use pytest_configure instead of import directly

### PR DESCRIPTION
**Pull request by janiversen**
Based on late review comments, contest is changed to use pytest_configure.

### CODE REVIEW

**Comment by janiversen** [comment]
@dhoomakethu I think this is what you wanted ?

**Review by dhoomakethu — APPROVED** [review]

File: pymodbus/client/asynchronous/serial.py [hunk 1]
```diff
@@ -38,6 +38,9 @@ def _framer(cls, method):
 
         raise ParameterException("Invalid framer method requested")
 
+    def __iter__(self):
```
Comment (dhoomakethu) [inline pymodbus/client/asynchronous/serial.py:41]: Why do we have to make `AsyncModbusSerialClient` an iterable ?

Comment (janiversen) [reply pymodbus/client/asynchronous/serial.py:41]: Without the __iter__ SonarCloud reported 2 bugs and asked that we use an iterable object, therefore I added __iter__

Actually sometimes I do not understand SonarCloud, I think it reports a lot of false positives.

Comment (dhoomakethu) [reply pymodbus/client/asynchronous/serial.py:41]: Thats a false positive , it assumes it requires an iterable for having the variable `loop` in the examples.
image

Comment (dhoomakethu) [reply pymodbus/client/asynchronous/serial.py:41]: Please remove that, I will mark that as false positivie

Comment (janiversen) [reply pymodbus/client/asynchronous/serial.py:41]: Yes it is, but I cannot find a away to disable the specific line, expected to be able to do something like #sonarcloud: disable=iter but it seems sonar cloud do not support such features.

Comment (janiversen) [reply pymodbus/client/asynchronous/serial.py:41]: I will remove it, but please tell me how to mark it as a false positive.

**Pull request merged**